docs/manuals/core/config/pipeline-config.md

```diff
@@ -2,7 +2,7 @@
 
     Note that `PipelineConfig^` has been deprecated in version 2.3 and will be combined with `ScenarioConfig^` in the following major version.
 
-A pipeline configuration is necessary to instantiate a [Pipeline](../concepts/pipeline.md). To create a
+A pipeline configuration is necessary to instantiate a [Sequence](../concepts/sequence.md). To create a
```

Comment:
"pipeline configuration"??

Reply:
It's untouched territory 😅 Just barely started on this config, it should be in a different PR but I clumsily committed it anyway

Reply:
I will make changes to this file on a separate PR, sorry for the confusion